docs/use_cases/model_applications/s2s_mid_lat/UserScript_fcstGFS_obsERA_Blocking.py

```diff
@@ -1,178 +1,251 @@
-# using stat_analysis.
-#
+# Atmospheric blocking is associated with extreme weather events.  This use case computes
+# atmospheric blocking events using the methodology in Miller & Wang (2019, 2022), which
+# identifies blocks from 500 mb height.  Various studies (Masato et al. 2013; Kitano and 
+# Yamada 2016) have suggested that 500 mb height produces a similar blocking climatology as 
+# results from potential temperature on a 2-PVU surface.
+#
+# The methodology in Miller & Wang (2019, 2022) first computes the Central Blocking Latitude 
+# (CBL) or storm track.  Allowing for an offset north and south of the storm track, reversals 
+# in geopotential height are then identified as Instantaneously Blocked longitudes (IBLs).  These 
+# IBLs are grouped when consective longitudes are blocked (GIBLs) and then blocks are identified 
+# by applying thresholds to ensure the large-scale, quasi-stationary characteristics of blocking 
+# anticyclones are met.  The IBLs, GIBLs, and blocks are computed separately for the model and 
+# observations, and contingency table statistics are computed to compare model performance at 
+# identifying IBLs and blocks.  The original code for computing blocking came from Douglas Miller.
+#
+#  * Miller, D. E., and Z. Wang, 2019a: Skillful seasonal prediction of Eurasian winter blocking and extreme temperature frequency. Geophys. Res. Lett., 46, 11 530–11 538, https://doi.org/10.1029/2019GL085035.
+#  * Miller, D. E., and Z. Wang, 2022: Northern Hemisphere Winter Blocking: Differing Onset Mechanisms across regions. J. Atmos. Sci., 79, 1291-1309, https://doi.org/10.1175/JAS-D-21-0104.1.
+#  * Masato, G., B. J. Hoskins, and T. J. Woollings, 2013: Winter and summer Northern Hemisphere blocking in CMIP5 models. J. Climate, 26, 7044–7059, https://doi.org/10.1175/JCLI-D-12-00466.1.
+#  * Kitano, Y., and T. J. Yamada, 2016: Relationship between atmospheric blocking and cold day extremes in current and RCP8.5 future climate conditions over Japan and the surrounding area. Atmos. Sci. Lett., 17, 616–622, https://doi.org/10.1002/asl.711.
 
 ##############################################################################
-# Datasets
-# --------
-#
-#  * Forecast dataset: GFS Forecast 500 mb height. 
-#  * Observation dataset: ERA Reanlaysis 500 mb height.
+# Version Added
+# -------------
 #
+# METplus version 4.0.0
 
 ##############################################################################
-# External Dependencies
-# ---------------------
-#
-# You will need to use a version of Python 3.6+ that has the following packages installed::
-#
-# * numpy
-# * netCDF4
-# * datetime
-# * bisect
-# * scipy
+# Datasets
+# --------
 #
-# If the version of Python used to compile MET did not have these libraries at the time of compilation, you will need to add these packages or create a new Python environment with these packages.
+# **Forecast dataset:** GFS Forecast 500 mb height for DJF 2000 - 2017 
```

Comment:
```suggestion
# **Forecast:** GFS Forecast 500 mb height for DJF 2000 - 2017 
```
Matching the template. "datasets" is redundant since it's in the Datasets section.